Enterprise-CMCS/macpro-mdct-mcr
# Sanction Error Bug

## Summary
When user selects “No” for question D3.VIII.8 in the sanction details page, the card does not show the "No" selection.

### Description

The entity was looking at only the date response but needed to look at the radio response as well, added an extension to look at the radio button if no date was selected.

### Related ticket

MDCT-2321

### How to test

1. Open a report on MCPAR
2. Select VIII: Sanctions from the side menu (expand D: Plan-Level Indicators)
3. Add a new Sanction
4. Edit sanction details
5. On question, D3.VIII.8 Remediation date non-compliance was corrected
6. Select No and finish filling the card info and save.
7. Sanction details should now be blue and Remediation date non-compliance was corrected will say no.


### Important updates

N/A

### Author checklist
